TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8283,4 +8291,4 @@ new_column_names: LeftParenthesisSeparator new_column_name (SQL_CommaSeparator n
 new_column_name: UserDefinedWord;
 // ------------------------------
 // End of DB2 coprocessor
-// ------------------------------
+// ------------------------------
```

Comment:
Why this change ? Please revert.

Reply:
The file had originally an empty line at its end, you removed it.
While it has absolutely no effect on the resulting code, this is a useless  change, I'd prefer to avoid those so please restore the empty line at end.